modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminContent.tsx

```diff
@@ -240,21 +238,32 @@ const TranslationAdminContent = ({
 			/>
 		</>
 	);
-};
+}
 
-TranslationAdminContent.propTypes = {
-	activeLanguageIds: PropTypes.arrayOf(PropTypes.string),
-	ariaLabels: PropTypes.shape({
-		default: PropTypes.string,
-		manageTranslations: PropTypes.string,
-		managementToolbar: PropTypes.string,
-		notTranslated: PropTypes.string,
-		tranlated: PropTypes.string,
-	}),
-	availableLocales: PropTypes.arrayOf(PropTypes.object).isRequired,
-	defaultLanguageId: PropTypes.string.isRequired,
-	lastDeletedLocale: PropTypes.object,
-	translations: PropTypes.object,
-};
+interface IProps extends Translations {
```

Comment:
Can we move this interface above where it is referenced? So above line 26 This will just keep things consistent across portal.

Reply:
~Also, what is the reason for extending the `Translations` interface? Why not just combine it all into one?~

nvm I see what you did